src/commands/index.ts

```diff
@@ -59,48 +59,68 @@ function toggleAngularPresetCommand() {
   const preset = 'angular';
   const value = getToggleValue(preset);
   const message = `${msg.ngPresetMessage} ${value ? msg.enabled : msg.disabled}. ${msg.restart}`;
-  togglePreset(preset, value, false);
-  showCustomizationMessage(message, [{ title: msg.reload }], applyCustomization, cancel, preset, !value, false);
+  const values = getConfig().inspect(`vsicons.presets.${preset}`);
```

Comment:
I think that by extracting this piece of code into a function will make it more clear and understandable. People will know that commands are sharing common functionality just by looking at the name of the new created function. Now they have to compare the several lines of each command to notice it. I vote for that refactor! 😜